Mathlib/Analysis/Fourier/LpSpace.lean

```diff
@@ -0,0 +1,91 @@
+/-
+Copyright (c) 2025 Moritz Doll. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Moritz Doll
+-/
+module
+
+public import Mathlib.Analysis.Distribution.FourierSchwartz
+public import Mathlib.Analysis.Normed.Operator.Extend
+
+/-!
+
+# The Fourier transform on L^p
+
+In this file we define the Fourier transform on `L2` as a linear isometry equivalence.
+
+## Main definitions
+
+* `Lp.fourierTransformLI`: The Fourier transform on `L2` as a linear isometry equivalence.
```

Comment:
I think this is normally how we write this throughout the library (do you have a counterexample?). I understand it's a little annoying that there is unicode in the identifier, but if we want to change it, I think a discussion on Zulip is warranted. I would prefer not to hold up the PR on account of a Zulip discussion though, so if you want to argue it, I suggest changing to `ₗᵢ` for the moment.
```suggestion
* `Lp.fourierTransformₗᵢ`: The Fourier transform on `L2` as a linear isometry equivalence.
```